controllers/user.js

```diff
@@ -7,7 +7,7 @@ const { Users } = models;
 const secretOrPrivateKey = process.env.SECRET;
 
 function jwtSignUser(payload) {
-  const ONE_WEEK = 60 * 60 * 24 * 7;
+  const ONE_WEEK = 60;
```

Comment:
I think this should still be left as one week, or at least a day. One minute for token expiration seems an overkill. And yes, we might overlook testing for the token expiration for now so as not to delay the work any more than we already have.